### ISSUE TITLE
[Bug]: internal error: sql query no record execution plan

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#5355]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 73c6715f
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

mysql> use system;
Reading table information for completion of table and column names
You can turn off this feature to get a quicker startup with -A

Database changed
mysql> select * from statement_info limit 10;
ERROR 20101 (HY000): internal error: the input value '{"code":200,"message":"sql query no record execution plan","steps":null,"success":false,"uuid:"21acc947-3fcb-11ed-a8df-98e872e2f140"}' is not json type for column 17

**Comment by xzxiong** [comment]
fix in https://github.com/matrixorigin/matrixone/pull/5338

**Comment by xzxiong** [comment]
fix in commit d3866ea5
